tests/test_features/test_manifold.py

```diff
@@ -166,49 +202,70 @@ def test_manifold_regression(self):
         )
 
         oz = Manifold(manifold="tsne", target="continuous", random_state=1)
-        assert not hasattr(oz, 'range_')
-
-        oz.fit(X, y)
+        assert not hasattr(oz, "range_")
 
-        assert not hasattr(oz, 'classes_')
-        assert hasattr(oz, 'range_')
+        oz.fit_transform(X, y)
+        oz.finalize()
+        assert not hasattr(oz, "classes_")
+        assert hasattr(oz, "range_")
         self.assert_images_similar(oz, tol=1.5)
 
+    def test_manifold_regression_3d(self):
+        """
+        Image similarity test for regression dataset (continuous y)
+        """
+        X, y = make_regression(
+            n_samples=300, n_features=7, n_informative=4, random_state=87
+        )
+
+        oz = Manifold(
+            manifold="tsne", target="continuous", random_state=1, projection=3
+        )
+        assert not hasattr(oz, "range_")
+
+        oz.fit_transform(X, y)
+        oz.finalize()
+        oz.cbar.set_ticks([])
+        assert not hasattr(oz, "classes_")
+        assert hasattr(oz, "range_")
+        self.assert_images_similar(oz, tol=15)
+
     def test_manifold_single(self):
```

Comment:
Could we also add a `test_manifold_single_3d`?

Reply:
Sure!